client/my-sites/plans/jetpack-plans/product-lightbox/constants.ts

```diff
@@ -1,7 +1,15 @@
-export const PRODUCT_OPTIONS = {
-	Backup: [
-		{ id: 'jetpack_backup_t0_yearly', answerText: '1GB' },
-		{ id: 'jetpack_backup_t1_yearly', answerText: '10GB' },
-		{ id: 'jetpack_backup_t2_yearly', answerText: '1TB(1000GB)' },
-	],
+import {
+	PRODUCT_JETPACK_BACKUP_T2_YEARLY,
+	PRODUCT_JETPACK_BACKUP_T1_YEARLY,
+} from '@automattic/calypso-products';
+import { translate } from 'i18n-calypso';
+
+export const PRODUCT_OPTIONS: Record< string, string > = {
+	[ PRODUCT_JETPACK_BACKUP_T1_YEARLY ]: translate( '10GB' ),
+	[ PRODUCT_JETPACK_BACKUP_T2_YEARLY ]: translate( '1TB(1000GB)' ),
```

Comment:
Jetpack Security is currently not showing the "Storage options".  I believe we need to add `PLAN_JETPACK_SECURITY_T1_YEARLY` and `PLAN_JETPACK_SECURITY_T2_YEARLY` into here, so that Jetpack Security will show "storage options:" also.  (See image)

![Markup 2022-09-13 at 16 37 12](https://user-images.githubusercontent.com/11078128/190005632-1a74467b-e07c-4171-8e08-3b5f2ba0d261.png)